src/test/java/com/checkmarx/flow/cucumber/integration/end2end/genericendtoend/BugTracker.java

```diff
@@ -100,11 +105,12 @@ void deleteIssues() {
                 Set<String> fields = new HashSet<>();
                 fields.addAll(
                         Arrays.asList("key", "project", "issuetype", "summary", "labels", "created", "updated", "status"));
-                Promise<SearchResult> temp = searchClient.searchJql(jqlQuery, 10, 0, fields);
-                SearchResult result = temp.get(500, TimeUnit.MILLISECONDS);
+                //Promise<SearchResult> temp = searchClient.searchJql(jqlQuery, 10, 0, fields);
```

Comment:
Because we have code for both cloud and on-premise (where existing SDK is being used), keep both tests.